modules/react/combobox/stories/Combobox.stories.mdx

```diff
@@ -0,0 +1,44 @@
+import {SymbolDoc, Specifications} from '@workday/canvas-kit-docs';
+
+import {Combobox} from '@workday/canvas-kit-react/combobox';
+import {Autocomplete} from './examples/Autocomplete';
+
+<Meta title="Features/Combobox" component={Combobox} />
+
+# Combobox
+
+Combobox is an abstract compound component. The term "Combobox" is based on the
+[Combobox Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/combobox/) as defined in the ARIA
+Authoring Practices Guide (APG):
+
+> A [combobox](https://w3c.github.io/aria/#combobox) is an input widget with an associated popup
+> that enables users to select a value for the combobox from a collection of possible values.
+
+Autocomplete and Select are examples of a Combobox and these components should be used instead.
```

Comment:
This implies Canvas Kit has Autocomplete and Select components that use Combobox, which we don't (yet). What do you think about this instead:

```suggestion
Combobox is not a standalone component and must be used in conjunction with other pieces to create
something usable.
```

Open to suggestions. "Pieces" is also intentionally vague, but open to alternatives there as well.

Reply:
We updated the overall intro to combobox. Can you re-review this section?